src/components/appNavigation/stackNavigation.js

```diff
@@ -146,20 +200,75 @@ class AppView extends Component<{ descriptors: any, navigation: any, navigationC
     }
     const Component = getComponent(descriptor.getComponent(), { screenProps })
     const pageTitle = title || activeKey
+    const menu = <SideMenuPanel screenProps={screenProps} />
     return (
       <React.Fragment>
         <Helmet>
           <title>{`Good Dollar | ${pageTitle}`}</title>
         </Helmet>
         {!navigationBarHidden && <NavBar goBack={backButtonHidden ? undefined : this.pop} title={pageTitle} />}
-        <ScrollView contentContainerStyle={scrollableContainer}>
-          <SceneView navigation={descriptor.navigation} component={Component} screenProps={screenProps} />
-        </ScrollView>
+        <SideMenu menu={menu} menuPosition="right" isOpen={store.get('sidemenu').visible}>
```

Comment:
can we use a local state variable to control the visibility?
use global (store) only for well global data, required by different components.